docs/keto/guides/rbac.mdx

```diff
@@ -2,6 +2,201 @@
 title: Role Based Access Control (RBAC)
 ---
 
-This guide will explain how to implement RBAC using Ory Keto. It's still work in
-progress. If you happened to figure this out yourself, please open a PR to add
-your findings on this page.
+This guide will explain how to implement RBAC using Ory Keto.
+
+:::warning
+
+Implementing RBAC currently is possible, but requires some workarounds. This
+guide enables RBAC support with Keto but native support is still work in
+progress. Follow the progress in
+[this issue](https://github.com/ory/keto/issues/598)
+
+:::
+
+[Role Based Access Control (RBAC)](https://en.wikipedia.org/wiki/Role-based_access_control)
+maps subjects to roles and roles to permissions. The goal of (H)RBAC is to make
+permission management convenient by grouping subjects in roles and assigning
+permissions roles. This type of access control is common in web applications
+where one often encounters roles such as "administrator", "moderator", and so
+on.
+
+In **Hierarchical Role Based Access Control (HRBAC)** roles can inherit
+permissions from other roles. The "administrator" role, for example, could
+inherit all permissions from the "moderator" role. This reduces duplication and
+management complexity around defining privileges.
+
+Let's come back to Alice, Bob, Peter, and blog posts and the matrix from the ACL
+example. This time we model the access rights using (H)RBAC and the roles
+`reader`, `author`, and `admin`:
+
+The role `admin` inherits all privileges from `author`, which inherits from
+`reader`. Only `Alice` (or rather her role `admin`) can delete blog posts,
+whereas `author` can create and modify blog posts.
```

Comment:
I think it is a bit confusing to have two examples in this one guide. We should rather use one from start to finish.